modules/persistent-storage-csi-vsphere-install-issues.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+[id="persistent-storage-csi-vsphere-install-issues_{context}"]
+= Removing an upstream vSphere CSI Operator Driver
+
+If you have installed an upstream vSphere CSI Operator Driver, and then attempt to update your cluster to {product-title} 4.10 or later, {product-title} prevents the update.
+
+To correct this issue and udpate the cluster, a cluster admin must:
+
+. Delete the upstream vSphere CSI driver Deployment object.
+. Delete the upstream vSphereCSI driver Daemonset objects:
+.. Be sure to delete `CSIDriver`:
++
+[output, screen]
+----
+~ $ oc delete CSIDriver csi.vsphere.vmware.com
+csidriver.storage.k8s.io "csi.vsphere.vmware.com" deleted
+----
++
+.. Be sure to delete configmap, and secret objects that were installed previously for the functioning of the upstream vSphere CSI Operator Driver.
```

Comment:
```suggestion
.. Be sure to delete the configmap and secret objects that were installed with of the upstream vSphere CSI Operator Driver.
```
Does this change the intended meaning too much? In either case, I think we can ditch the comma because the both the config map and secret objects were previously installed. WDYT?

Reply:
Will customers know where to find the _secret objects_?